dts/bindings/sensor/hamamatsu,s11059.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022, Hiroki Tada
+# SPDX-License-Identifier: Apache-2.0
+
+description: HAMAMATSU PHOTONICS s11059 color sensor.
```

Comment:
Fix capatalization here.  

Also.  Maybe make this:
```
description: |
     Hamamatsu Photonics S11059 Color Sensor.  See datasheet at
     https://datasheetspdf.com/pdf/1323325/Hamamatsu/S11059-02DT/1
```

Reply:
fixed it